infrastructure/tfvars.json

```diff
@@ -1,25 +1,23 @@
 {
   "location": "West Europe",
-  "terraform_state_storage_account": "agidtfsaprd",
-  "terraform_state_storage_container": "agidtfscprd",
   "cosmosdb_failover_location": "North Europe",
-  "azurerm_resource_group": "agidrgprd",
-  "azurerm_storage_account": "agidsa",
-  "azurerm_storage_container": "agidscprd",
+  "azurerm_resource_group": "agid-rg-prod",
+  "azurerm_storage_account": "agidstorageprod",
```

Comment:
better naming, this one needs dashes too

Reply:
they can't have dashes, only alphanumeric lowercase characters.